hibike/devices/YogiBear/motor.cpp

```diff
@@ -4,6 +4,7 @@
 //tab to handle all controls issued to the motor including driving and braking
 
 bool motorEnabled = false;
+float deadBand = 0;
```

Comment:
Sounds reasonable to me. I believe doug wanted it to default to 0, but I think in this case we will change it to 0.05 so that by default the controller in its resting position won't drive the motor because of a small tilt.